.pipelines/.vsts-vhd-builder-windows.yaml

```diff
@@ -169,6 +170,8 @@ jobs:
       -e OS_NAME="Windows" \
       -e SKU_NAME=${SKU_NAME} \
       -e OFFER_NAME="Windows" \
+      -e IMAGE_VERSION=${WINDOWS_VERSION} \
+      -e HYPERV_GENERATION="V1" \
```

Comment:
What is the meaning for Windows?

Reply:
It's  Hyper-V Generation 1, like, We are using `2019-Datacenter-with-Containers-smalldisk` instead of `2019-datacenter-with-containers-smalldisk-g2`, so set hyper-generations to v1.
@jsturtevant told me that we don't support v2 images, but cannot remember the detail.